core/src/components/spinner/spinner.ionic.scss

```diff
@@ -4,6 +4,10 @@
 // Ionic Spinner
 // --------------------------------------------------
 
+:host {
+  --color: #{globals.$ionic-color-neutral-800};
```

Comment:
Updated this to match Figma else it was trying to re-generate with black being the default value.